api/python/cellxgene_census/src/cellxgene_census/_release_directory.py

```diff
@@ -25,14 +25,27 @@
         "s3_region": Optional[str],  # [deprecated: only used in census < 1.6.0] if an S3 URI, has optional region
     },
 )
+CensusVersionRetraction = TypedDict(
+    "CensusVersionRetraction",
+    {
+        "date": str,  # the date of retraction
+        "reason": Optional[str],  # the reason for retraction
+        "info_url": Optional[str],  # a permalink to more information
+        "replaced_by": Optional[str],  # the census version that replaces this one
+    },
+)
+RELEASE_FLAGS = {"lts", "retracted"}
+ReleaseFlag = Literal["lts", "retracted"]
```

Comment:
this seems brittle/not-DRY, as the code later relies on the type definition always matching the set definition.

Suggest:
```
ReleaseFlag = Literal["lts", "retracted"]
RELEASE_FLAGS = { *typing.get_args(ReleaseFlag) }
```

Reply:
ah yes, was hoping to DRY this up, done!